homeassistant/components/isy994/const.py

```diff
@@ -133,14 +126,29 @@
 KEY_ACTIONS = "actions"
 KEY_STATUS = "status"
 
-PLATFORMS = [BINARY_SENSOR, SENSOR, LOCK, FAN, COVER, LIGHT, SWITCH, CLIMATE]
-PROGRAM_PLATFORMS = [BINARY_SENSOR, LOCK, FAN, COVER, SWITCH]
+PLATFORMS = [
+    Platform.BINARY_SENSOR,
+    Platform.SENSOR,
+    Platform.LOCK,
+    Platform.FAN,
+    Platform.COVER,
+    Platform.LIGHT,
+    Platform.SWITCH,
+    Platform.CLIMATE,
```

Comment:
```suggestion
    Platform.BINARY_SENSOR,
    Platform.CLIMATE,
    Platform.COVER,
    Platform.FAN,
    Platform.LIGHT,
    Platform.LOCK,
    Platform.SENSOR,
    Platform.SWITCH,
```

Reply:
Changing the order made this a bit harder to check.  At least for me it would have been preferred to do it in a followup PR